### DISCUSSION TITLE
Add port to the wizard's rtmp url for second instance

### DISCUSSION THREAD

**Discussion by MauricioOnetto** [datarhei/restreamer#463] [Ideas]
Hey people

For second (or more) instance we need to change the default rtmp port (1935) for a custom.

Wizard propose the rtmp url without port, so the server asumes that the port is 1935.

I've being solveing this manually adding my custom port to the wizard's url.

Is it possible that Restreamer detect the custom port and automatic add it to the wizard?

I think it will be useful.

**Comment by jstabenow** [comment]
Hey @MauricioOnetto
1. Change the RTMP port:     
    [docs.datarhei.com/restreamer/knowledge-base/manual/system-settings/rtmp](https://docs.datarhei.com/restreamer/knowledge-base/manual/system-settings/rtmp)
2. Launch the Restreamer with your new port-mapping:
   Old: `docher run {...param} -p 1935:1935 {...param}`    
   New: `docker run {...param} -p CUSTOM_PORT:CUSTOM_PORT {...param}`

**Example:**
1. Change the RTMP port (e.g. 2000):
 ![Bildschirmfoto 2022-11-15 um 21 28 04](https://user-images.githubusercontent.com/7462030/202019432-4200ad07-2470-404f-a5dd-bee44effda90.png)

3. Map the port: 
    `docker run {...param} -p 2000:2000 {...param}`

5. Launch the wizard:
![Bildschirmfoto 2022-11-15 um 21 29 06](https://user-images.githubusercontent.com/7462030/202019448-908cbff9-b9c3-42a0-9466-156d676c0dff.png)

**Reply by MauricioOnetto** [reply]
Thanks a lot!!!
Your answer solve my problem

I was (wrongly) using the next code
`docker run {...param} -p CUSTOM_PORT:1935 {...param}`

Thank you @jstabenow
